bin/ci/deploy-dockerhub.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# THIS IS MEANT TO BE RUN BY CI
+
+set -e
+
+# Usage: retry MAX CMD...
+# Retry CMD up to MAX times. If it fails MAX times, returns failure.
+# Example: retry 3 docker push "$DOCKERHUB_REPO:$TAG"
+function retry() {
```

Comment:
@autrilla please take a look at https://github.com/mozilla/fxa-oauth-server/pull/554/files#diff-1d37e48f9ceff6d8030570cd36286a61R42 We have a simplified version of container creation.

We never really had problems pushing to DockerHub, so after chatting with @jrgm we don't really want to complicate this. Especially because we would have to create this script in all of our repos.

Reply:
I guess in this case, this repo can use the `python` circle ci tag, not `node`

Reply:
Other projects have had trouble, @milescrabill introduced this iirc. I agree that there's a lot of code duplication and we should probably put this some place standard.